posthog/errors.py

```diff
@@ -152,7 +152,11 @@ class CHQueryErrorTooManySimultaneousQueries(InternalCHQueryError):
     50: ErrorCodeMeta("UNKNOWN_TYPE"),
     51: ErrorCodeMeta("EMPTY_LIST_OF_COLUMNS_QUERIED"),
     52: ErrorCodeMeta("COLUMN_QUERIED_MORE_THAN_ONCE"),
-    53: ErrorCodeMeta("TYPE_MISMATCH", user_safe=True),
+    53: ErrorCodeMeta(
+        "TYPE_MISMATCH",
+        user_safe="One or more property values in your filters are in an incorrect format."
```

Comment:
This needs to stay as boolean.

Reply:
https://github.com/PostHog/posthog/blob/bb997bcf248723968575c40ab60ea7fd9df58fde/posthog/errors.py#L36-L39

https://github.com/PostHog/posthog/blob/bb997bcf248723968575c40ab60ea7fd9df58fde/posthog/errors.py#L303-L306


@webjunkie I observed that `user_safe` is being assigned a string value in a few other scenarios, which made me think this might be the ideal way to handle it. I'll go ahead and change `user_safe` to a boolean as suggested


Can you please let me know the best way to provide a more descriptive error message in that case? 
Should I handle this by overriding the message in the SessionRecordingViewSet where the ValidationError is raised, or is there another approach you'd recommend?

TIA

CC: @daibhin

Reply:
I think we will want to capture the error on the frontend and display it there. If you look at the `frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts` file you should be able to add a new listener for `loadSessionRecordingsFailure` and maybe call `lemonToast.error` with the user safe string

Reply:
Got it, thanks! Will give it a try

Reply:
> I think we will want to capture the error on the frontend and display it there. If you look at the `frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts` file you should be able to add a new listener for `loadSessionRecordingsFailure` and maybe call `lemonToast.error` with the user safe string

Have pushed this change now via commit: https://github.com/PostHog/posthog/pull/24300/commits/b9e12573b2aa93cc3f9d641b48d73394ac47c6ce